src/main/kotlin/nl/avisi/kotlinwebtest/jdbc/dsl/Dsl.kt

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
Great idea. Missing some tests for this functionality, though.

Reply:
fixed